mod_reforged/hooks/entity/world/attached_location/surface_iron_vein_location.nut

```diff
@@ -4,7 +4,7 @@
 	{
 		switch (_id)
 		{
-			case "building.marketplace":
+			case "building.weaponsmith":
```

Comment:
This addition is outside of the scope of this PR. It should be added in its own branch.